_layouts/page.html

```diff
@@ -62,11 +62,9 @@
       {% endunless %}
       <article>
         {% if page.title %}
-          {% unless page.title_hidden %}
```

Comment:
Totally fine with removing this again since it is not needed at the moment. But please decrease the indentation for the inner three lines.